src/__test__/addRemove.test.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import { addItem, removeItem } from '../todoList.js';
+
+describe('addItem', () => {
+  beforeEach(() => {
+    localStorage.clear();
+  });
+  test('should add a task item to the todo list', () => {
+    const task = 'Do laundry';
+    addItem(task);
+    const storedList = JSON.parse(localStorage.getItem('todoList'));
+    expect(storedList).toHaveLength(1);
+    expect(storedList[0]).toMatchObject({
+      description: task,
+      completed: false,
+    });
+  });
+
+  test('should not add an empty task item to the todo list', () => {
+    addItem('');
+    const storedList = JSON.parse(localStorage.getItem('todoList'));
+    expect(storedList).toBeNull();
+  });
+});
+describe('Remove item', () => {
+  test('should not remove a task item if index is invalid', () => {
+    const task = 'Do laundry';
+    addItem(task);
+    removeItem(1);
+    const storedList = JSON.parse(localStorage.getItem('todoList'));
+    expect(storedList).toHaveLength(1);
+    expect(storedList[0]).toMatchObject({
+      description: task,
+      completed: false,
+    });
+  });
```

Comment:
- [ ] @Innocent259 and @SanaTameem great work adding tests for the remove function, however according to the project instruction as shown below, were are supposed to do a test to show we can remove one element from the array of tasks, kindly consider adding another test
![image](https://github.com/Innocent259/Todo-List/assets/61408860/b2e160a0-aca9-4073-8b72-4238f843717a)